components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -108,7 +119,16 @@
             (ConfigurationContext) config.getServletContext().getAttribute(CarbonConstants.CONFIGURATION_CONTEXT);
     UserStoreConfigAdminServiceClient userStoreConfigAdminServiceClient = new UserStoreConfigAdminServiceClient(cookie, backendServerURL, configContext);
     classApplies = userStoreConfigAdminServiceClient.getAvailableUserStoreClasses();
-    Iterator<String> iterator = UserStoreMgtDataKeeper.getAvailableDomainNames().iterator();
+    UserStoreDTO[] userStoreDTOs;
+    Map<String, Map<String, String>> userStoreManagers = new HashMap<String, Map<String, String>>();
+    userStoreDTOs = userStoreConfigAdminServiceClient.getActiveDomains();
+    if (userStoreDTOs[0] != null) {
```

Comment:
Can userStoreDTOs be null? If so if condition should be changed to,
if (userStoreDTOs != null) {

and handle elements being null in as following comment